### PR TITLE
feat: allow for some params to not be expanded

### PR DESCRIPTION
**Pull request by FedeAbella**
This allows some extra verbosity at the moment of using `addParams` to `cmd_builder` to be used so that particular parameter value will **not** be expanded when the cmd is built. It's done so as to keep all the other method calls the same and working as intended.

### CODE REVIEW

**Comment by xixiaofinland** [comment]
The design looks fine to me. But I suggest you adding expanding/no-expanding tests to guard this new feature if not a big hassle. 

Or I can add it afterwards, which is also fine. @FedeAbella

**Comment by FedeAbella** [comment]
Absolutely! Don't worry abou it, I'll take care of it. I needed an excuse to learn how tests work in lua (and in this project especifically) anyways :rofl:

**Comment by FedeAbella** [comment]
@xixiaofinland could use your help here: I seem to be running into an issue running the tests with `mini.test` where it can't find the treesitter parsers module:
image

Even though I do have it installed. Any idea what might be going on?

**Comment by FedeAbella** [comment]
Ah, I definitely missed that. Thanks!

**Comment by FedeAbella** [comment]
Added some tests @xixiaofinland. Let me know if they look enough for you. I've also taken your advice and moved the whole thing to a new `addParamsNoExpand` method.

I think I kind of got the gist of the tests, so I'll see about expanding some other tests later if these look good to you

File: tests/test_cmd_builder.lua [hunk 1]
```diff
@@ -72,6 +72,31 @@ T["setup()"]["can supply mutliple params in table in addParams()"] = function()
...
 end
 
+T["setup()"]["can supply flag:value string in addParamsNoExpand(), and will not be expanded"] = function()
+  child.open_in_sf_dir("test.txt")
+  local result = child.lua_get('B:new():cmd("apex"):act("run"):addParamsNoExpand("-f", "%:p"):build()')
+  local expected = 'sf apex run -f "%:p" -o "t_org"'
+  eq(result, expected)
+end
+
+T["setup()"]["can supply flag:value string in more than one addParamsNoExpand, and none will be expanded()"] = function()
```
Comment (xixiaofinland) [inline tests/test_cmd_builder.lua:82]: These tests don't have `addParamsNoExpand()` in test setup but the description says so. Something wrong? @FedeAbella

Comment (FedeAbella) [reply tests/test_cmd_builder.lua:82]: Sorry about that, I was multitasking, got one of the tests to work by setting the file, then forgot to apply that to the other ones :( It's fixed now

**Pull request merged**